js/client.js

```diff
@@ -2509,13 +2509,13 @@ function toId() {
 			var name = data.name;
 			var avatar = data.avatar || '';
 			var group = ((Config.groups[data.roomGroup] || {}).name || '');
```

Comment:
Sorry to make you do this, but could you also rename `group` to `groupName`?

This is already a lot more readable, though, so thanks for that! It might seem minor to you, but it's a huge difference in readability to me.

Reply:
OK.